README.md

```diff
@@ -1,2 +1,6 @@
 # ML-python-sklearn-rc-cpu-training
-Jenkins X quick start for training and deploying Ridge Classification using the SciKit-Learn library
+
+
+Quickstart project for executing a Wine classifier using the SciKit-Learn framework on a CPU.
+
+This Quickstart trains the model and persists as in ONNX format. The service runtime will then serve the model on localhost where the user can then send GET requests to perform inference.
```

Comment:
Maybe add more info about MLOps so the user understands reasoning for some of the code (ie threshold) https://github.com/mxdillon/ML-python-tensorflow-mnist-cpu-training/blob/master/README.md